src/messaging/ExchangeContext.h

```diff
@@ -128,7 +128,11 @@ class DLL_EXPORT ExchangeContext : public ReliableMessageContext,
      * A notification that we will have SendMessage called on us in the future
      * (and should stay open until that happens).
      */
-    void WillSendMessage() { mFlags.Set(Flags::kFlagWillSendMessage); }
+    void WillSendMessage(bool isPeerActive = false)
```

Comment:
Adding an argument to the method that did not have one before, and that overloads usage, is dangerous. When is it that "isPeerActive" has to be set true/false? This is not explained at all in the method docs.

Whatever changes the behavior to be different than Matter 1.0/1.1/1.2 behavior has to be a new method name and the clients should be able to tell trivially when they have to call it. Please do not add an argument to `WillSendMessage`.

Reply:
resolving since the this changes doesn't exist anymor